src/types.ts

```diff
@@ -40,6 +40,19 @@ export type PackedClaims = {
 
 export type Hasher = (data: string) => Promise<string>;
 export type SaltGenerator = (size) => string;
+
+export interface IssueSDJWTOptions {
+  header?: JWTHeaderParameters;
+  payload: JWTPayload;
+  disclosureFrame: DisclosureFrame;
+  alg: string;
```

Comment:
@ragnika I think here we should as well have some sort of enum of most popular algs and do a verification of the param passed.